server/api/usersRoutes.js

```diff
@@ -1,5 +1,12 @@
 const router = require('express').Router()
-const {Review, User} = require('../db/models')
+const {
+  Review,
+  User,
+  UserOrder,
+  OrderProduct,
+  Product,
+  Brand
+} = require('../db/models')
 module.exports = router
 
 router.get('/', async (req, res, next) => {
```

Comment:
Is this the user route we discussed last night where we might not even need anymore?
I recall there was a discussion in our breakout room where having some user-related route was redundant since we weren't even planning on using that for our cart.
Let me know

Reply:
Sort of, I propose we leave this in for now, and discuss if we need to delete or repurpose